frontend/components/CompactNavigationBar.js

```diff
@@ -3,20 +3,19 @@ import { useRouter } from "next/router";
 import Link from "next/link";
 import Image from "next/image";
 import styles from "../styles/Navbar.module.css";
-
-// import menu icons
+import {
+  navigationBarContent,
+  aboutUsDropdownContent,
+  mediaDropdownContent,
+} from "../data/navbarData";
+import {
+  changeAboutUsToArrowDown,
+  changeAboutUsToArrowRight,
+  changeMediaToArrowDown,
+  changeMediaToArrowRight,
+} from "./navbarHelpers";
```

Comment:
Rename the file to `navbar.js` and move the file to the helpers folder (merge develop into this branch)

Reply:
donee !